doc/classes/DisplayServer.xml

```diff
@@ -2131,6 +2131,44 @@
 				Returns the current value of the given window's [param flag].
 			</description>
 		</method>
+		<method name="window_get_hdr_output_current_max_luminance" qualifiers="const">
+			<return type="float" />
+			<param index="0" name="window_id" type="int" default="0" />
+			<description>
+				Returns the current maximum luminance in nits (cd/m²) for HDR content for the window specified by [param window_id].
+				If max luminance is being auto adjusted based on the display's capabilities, this will return that value.
+				Otherwise, it will return the value set by [method window_set_hdr_output_max_luminance].
+				[b]Note:[/b] Requires support for [constant FEATURE_HDR_OUTPUT].
+			</description>
+		</method>
+		<method name="window_get_hdr_output_current_reference_luminance" qualifiers="const">
+			<return type="float" />
+			<param index="0" name="window_id" type="int" default="0" />
+			<description>
+				Returns the current reference luminance in nits (cd/m²) for HDR content for the window specified by [param window_id].
+				If reference luminance is being auto adjusted based on the display's capabilities, this will return that value.
+				Otherwise, it will return the value set by [method window_set_hdr_output_reference_luminance].
```

Comment:
```suggestion
				Returns the current reference luminance in nits (cd/m²) for HDR content for the window specified by [param window_id]. If the reference luminance is being automatically adjusted based on the display's capabilities, this will return that value. Otherwise, it will return the value set by [method window_set_hdr_output_reference_luminance].
```